skycoin/skywire
# Bandwidth received fix

Did you run `make format && make check`? yes

Fixes #767 

 Changes:	
- Fixes total bandwidth received 

How to test this PR:

See #767 and run it as it is mentioned.


https://user-images.githubusercontent.com/72057554/135988241-928a38f7-756e-4db3-b613-5f7a12628bbd.mp4
